maths/harshad_numbers.py

```diff
@@ -0,0 +1,132 @@
+"""
+A Harshad number is divisible by the sum of its digits in any base n.
+Reference: https://en.wikipedia.org/wiki/Harshad_number
+"""
+
+
+def int_to_base(number: int, base_of_interest: int) -> str:
+    digits = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+    result = ""
+
+    while number > 0:

```

Comment:
What happens if the input is negative? If you only mean for this function to accept non-negative inputs, then make sure that this function raises an Exception for negative inputs.